culture/code-of-conduct.md

```diff
@@ -0,0 +1,129 @@
+
+:::{important}
+When in doubt, please {ref}`report <coc:reporting>` unacceptable behavior to us. If someone’s behavior outside of a 2i2c space makes you feel unsafe at 2i2c, that is absolutely relevant and actionable for us.
+:::
+
+## Enforcement
+
+We’ve categorized unacceptable behavior into abuse and unwelcoming behavior in the section above.
+
+If we witness or receive a report about abusive behavior, we will contact the perpetrator to have a conversation with them and verify what has transpired. We will [follow this response protocol](response-protocol.md).
+
+If we verify abusive behavior, they will be removed from the 2i2c community. Their Slack account will be deactivated, and permissions will be removed from any 2i2c-related repositories. They will not be welcome in any physical or digital spaces covered by the 2i2c Code of Conduct.
+
+If we verify unwelcome, but non-abusive behavior, we will have a conversation with the person so they understand the expectation that they not repeat the behavior a second time. See [the follow-up protocol for more information](coc:reporting:follow-up).
+ 
+[This is the protocol](response-protocol.md) that 2i2c Directors and Steering Council members will use to respond to reports of code of conduct violations.
+
+(coc:reporting)=
+## Reporting
+
+If you see a violation of our code of conduct, please [report it to the 2i2c Executive Director](https://docs.google.com/forms/d/e/1FAIpQLSeKFuoghGzftqqgaa0xllZyocdUJsWZmoSaFFlerimg4n_Y8A/viewform?usp=sf_link).
+
+
+### Why should I report?
+
+* **You are responsible for making 2i2c a safe and comfortable space for everyone.** Everyone in our community shares this responsibility. 2i2c Team and Steering Council members are not around the online spaces or at 2i2c events all the time, so we cannot enforce the code of conduct without your help.
+* **The consequences for the 2i2c community of not reporting bad behavior outweigh the consequences for one person of reporting it.** We sometimes hear “I don’t want X person to meet consequences because I told someone about their bad behavior.” Consider the impact on everyone else at 2i2c of letting their behavior continue unchecked.
+* 2i2c only works as an open, participatory, community-driven community because of shared trust between community members. **Reporting code of conduct violations helps us identify when this trust is broken, to prevent that from happening in the future.**
+
+
+### Where and how to report
+
+Please report all code of conduct violations using [our reporting form](https://docs.google.com/forms/d/e/1FAIpQLSeKFuoghGzftqqgaa0xllZyocdUJsWZmoSaFFlerimg4n_Y8A/viewform?usp=sf_link). This is a short Google Form that will be sent to the Executive Director. If you prefer, you may also [email the Executive Director directly](mailto:director@2i2c.org).
```

Comment:
Not that I expect this to happen, but we should probably have an alternative channel in case the incident itself involves the ED.

Reply:
I don't have a good solution here but I agree with Ryan. An incident involving the ED needs a reporting path. I have concerns about a different aspect though. The ED is naturally incentivized to protect and sustain 2i2c and there may emerge situations where that goal is in tension with proper resolution on an incident. Imagine some situations involving a talented 2i2c developer and a community member.... I can imagine an abstract ED (not Chris mind you...) choosing, perhaps wishing is a better word here, to minimize the incident to retain the developer leaving the community member harmed and with justice not being served. At a minimum, two people should receive the reports.

In situations where 2i2c may be serving another community, (e.g. OpenScapes using 2i2c infrastructure), we should find a way to integrate the organizers/leaders of the other community in the CoC management. Ideally, this integration should also build in the expectation that participants actually read the CoC document and behave accordingly. 

I was impressed by the way ADSA handled CoC.